Source/WebCore/platform/graphics/cg/GradientRendererCG.cpp

```diff
@@ -121,7 +155,7 @@ GradientRendererCG::Strategy GradientRendererCG::makeGradient(ColorInterpolation
         if (hasOnlyBoundedSRGBColorStops(stops)) {
             for (const auto& stop : stops) {
                 if (destinationColorSpace) {
-                    auto [r, g, b, a] = stop.color.toResolvedColorComponentsInColorSpace(*destinationColorSpace);
+                    auto [r, g, b, a ] = getResolvedColorComponentsInColorSpace(stop.color, *destinationColorSpace);
```

Comment:
Nit: don't need that space
```suggestion
                    auto [r, g, b, a] = getResolvedColorComponentsInColorSpace(stop.color, *destinationColorSpace);
```